src/legacy/core_plugins/vis_default_editor/public/components/sidebar/sidebar_title.tsx

```diff
@@ -0,0 +1,169 @@
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useCallback, useState } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiText,
+  EuiButton,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { Vis } from 'src/legacy/core_plugins/visualizations/public';
+import { SavedSearch } from '../../../../kibana/public/discover/np_ready/types';
+
+interface LinkedSearchProps {
+  savedSearch: SavedSearch;
+  vis: Vis;
+}
+
+interface SidebarTitleProps {
+  isLinkedSearch: boolean;
+  savedSearch?: SavedSearch;
+  vis: Vis;
+}
+
+export function LinkedSearch({ savedSearch, vis }: LinkedSearchProps) {
+  const [showPopover, setShowPopover] = useState(false);
+  const closePopover = useCallback(() => setShowPopover(false), []);
+  const onClickButtonLink = useCallback(() => setShowPopover(v => !v), []);
+  const onClickUnlikFromSavedSearch = useCallback(() => {
+    setShowPopover(false);
+    vis.emit('unlinkFromSavedSearch');
+  }, [vis]);
+
+  const linkButtonAriaLabel = i18n.translate(
+    'visDefaultEditor.sidebar.savedSearch.linkButtonAriaLabel',
+    {
+      defaultMessage: 'Link to saved search. Click to learn more or break link.',
+    }
+  );
+
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      className="visEditorSidebar__titleContainer visEditorSidebar__linkedSearch"
+      gutterSize="xs"
+      responsive={false}
+    >
+      <EuiFlexItem grow={false}>
+        <EuiIcon type="search" />
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false} className="eui-textTruncate">
+        <EuiTitle size="xs" className="eui-textTruncate">
+          <h2
+            title={i18n.translate('visDefaultEditor.sidebar.savedSearch.titleAriaLabel', {
+              defaultMessage: 'Saved search: {title}',
+              values: {
+                title: savedSearch.title,
+              },
+            })}
+          >
+            {savedSearch.title}
+          </h2>
+        </EuiTitle>
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false}>
+        <EuiPopover
+          anchorPosition="downRight"
+          button={
+            <EuiButtonIcon
+              aria-label={linkButtonAriaLabel}
+              data-test-subj="unlinkSavedSearch"
+              iconType="link"
+              onClick={onClickButtonLink}
+            />
```

Comment:
@sulemanof Actually, do you mind putting the tooltip back in with the same content as the aria-label? Sorry, I just realized that the original link icon, when clicked immediately would unlink the saved search. People might then be scared to click it now given that it's the only way to "go to discover"

Reply:
ok, done